_entities.md

```diff
@@ -125,11 +125,11 @@ name      | The display name of the contact created by joining the first and las
   "ask": "6420.05",
   "bid": "6419",
   "currency": "USD",
-  "pair": "BTCUSD"
+  "pair": "BTC-USD"
```

Comment:
Actually, the object returned by the API does not feature dash separation in the pair property. I imagine this was done not to cause any breaking changes.